### PR TITLE
docs/ewx-177 worker pallet on polkadot.js

### PR DESCRIPTION
**Pull request by JGiter**
Provides instructions how to communicate with Worker pallet in `polkadot.js`. This instructions will be updated as pallet development continues. For now instructions includes only signing up registrar and solution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,3 +135,129 @@ cargo update -p pallet-ewx-worker-solution
...
+
+## Example scenarios
+
+### Solution Registrar Flow
+
+1. Follow the instructions to [register as a solution registrar via Polkadot.js](#signing-up-solution-registrar)
+
+2. Use the follow script as an example to register solution.
+`ALICE` is one of the predefined development accounts. This script first
+signups ALICE as solution registrar, which allows next to register solution.
```
Comment (jrhender) [inline README.md:192]: As the script includes solution registrar sign-up, I think the first step is not needed now.
```suggestion
Use the follow script as an example to register solution.
`ALICE` is one of the predefined development accounts. This script first
signups ALICE as solution registrar, which allows next to register solution.
```

File: README.md [hunk 2]
```diff
@@ -135,3 +135,127 @@ cargo update -p pallet-ewx-worker-solution
...
+
+  const keyring = new Keyring({ type: "sr25519" });
+  const ALICE_KEYRING = keyring.addFromUri("//Alice", {
+    name: "Alice default",
+  });
+  const ALICE_ADDRESS = ALICE_KEYRING.address; // "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
+
+  const registrar_name = "Alice";
+  const registrar_legal_location = "Alice place";
+  await new Promise<void>(async (resolve) => {
```
Comment (JGiter) [inline README.md:208]: Not sure if this is the best way to ensure that registrar signed up before registering solution

Comment (jrhender) [reply README.md:208]: I'm not sure either. How does this code work?
```
await new Promise<void>(async (resolve) => {
    let unsub = await api.tx.workerNodePallet
      .signupSolutionRegistrar(registrar_name, registrar_legal_location)
      .signAndSend(ALICE_KEYRING, ({ status }) => {
        if (status.isFinalized) {
          unsub();
          resolve();
        }
      });
  });
```
Is the callback passed to `signAndSend` called multiple times (until `unSub`) is called?

By the way, I think full words are generally better than abbreviations as it is clearer, so I would replace `unSub` with `unSubscribe`

**Review by jrhender — APPROVED** [review]
At any rate, as instructions are clear and the script is working, I propose that this be merged 👍

File: README.md (hunk 2 above)
Comment (JGiter) [reply README.md:208]: Callback indeed called each time when extrinsic status has been changed. 
I'll rename `unsub`

Comment (JGiter) [reply README.md:208]: From logs I can see that this extrinsic passes Ready -> Broadcast -> InBlock -> Finalized

**Pull request merged**